packages/jaeger-ui/src/components/Monitor/ServicesView/serviceGraph.test.js

```diff
@@ -39,61 +39,66 @@ describe('<ServiceGraph>', () => {
     expect(wrapper.length).toBe(1);
   });
 
-  it('Loading indicator is displayed', () => {
-    expect(wrapper).toMatchSnapshot();
+  it('displays loading indicator when loading', () => {
+    expect(wrapper.find('LoadingIndicator').exists()).toBe(true);
```

Comment:
`exists()).toBe(true)` - is there a simpler form like isNotNull?

Reply:
I don't think. But there are similar from like ***`toBeTruthy()`*** or **`toBeTrue()`**

Reply:
https://github.com/jaegertracing/jaeger-ui/blob/60093a4ce45c6f14ace930386d1f19d37d8f9a36/packages/jaeger-ui/src/components/Monitor/ServicesView/index.test.js#L289

Reply:
Ahh OKK